### PR TITLE
Update document-issue-template.md

### PR DESCRIPTION
**Pull request by ji-hoooon**

### CODE REVIEW

**Review by kimjaewoo2233 — APPROVED** [review]
good

**Comment by ramloper** [comment]
good

File: .github/ISSUE_TEMPLATE/document-issue-template.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-about: Describe this issue template's purpose here.
-title: ''
-labels: ''
-assignees: ''
+### 제목
 
 ---
 
+### 내용
+
```
Comment (jungjongyeong) [inline .github/ISSUE_TEMPLATE/document-issue-template.md:6]: good

**Review by jungjongyeong — COMMENTED** [review]
good

**Review by ji-hoooon — COMMENTED** [review]
Approve 해주세요!@

File: .github/ISSUE_TEMPLATE/document-issue-template.md (hunk 1 above)
Comment (Junggi3466) [reply .github/ISSUE_TEMPLATE/document-issue-template.md:6]: good

File: .github/ISSUE_TEMPLATE/document-issue-template.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
----
-name: Document issue template
-about: Describe this issue template's purpose here.
-title: ''
-labels: ''
```
Comment (kimjaewoo2233) [inline .github/ISSUE_TEMPLATE/document-issue-template.md:5]: good

**Review by kimjaewoo2233 — APPROVED** [review]

**Review by jungjongyeong — APPROVED** [review]

**Review by ramloper — APPROVED** [review]
gooooooooooooood

**Review by Junggi3466 — APPROVED** [review]
good

**Pull request merged**